public/app/core/components/AppChrome/AppChrome.tsx

```diff
@@ -20,8 +20,10 @@ export function AppChrome({ children }: Props) {
   const styles = useStyles2(getStyles);
   const { chrome } = useGrafana();
   const state = chrome.useState();
+  const dashboard = useSelector((state) => state.dashboard.getModel());
+  const isEditorEmbedded = dashboard?.meta.isEditorEmbedded;
```

Comment:
no dashboard access should happen here in AppChrome :)

Reply:
Fixed.